generators/app/templates/_templates/block/block.html.twig

```diff
@@ -35,7 +35,7 @@
 <div{{ attributes.addClass(classes) }}>
   {{ title_prefix }}
   {% if label %}
-    <h2{{ title_attributes }}>{{ label }}</h2>
+    <h2{{ title_attributes.addClass('block__title') }}>{{ label }}</h2>
   {% endif %}
   {{ title_suffix }}
   {% block content %}
```

Comment:
@sagarrchauhan Shouldn't the content be wrapped in a `block__content` class?

Reply:
This is a bit tricky here. There are cases where we would want to have the wrapper inside the rendered content rather than having it in block itself or not to have the wrapper at all. This is to reduce the level of wrappers a component can have and thereby reducing the DOM depth. Let me know if you think otherwise.

Reply:
Changed